phylorthology/downsamp_test_tsar_samplesheet.csv

```diff
@@ -0,0 +1,7 @@
+species,file,taxonomy,shallow_db,broad_db,mode,uniprot,mcl_test
+Tetrahymena_thermophila,/home/ubuntu/environment/github/test-datasets/phylorthology/tsar/downsampled_proteomes/Tetrahymena_thermophila.fasta,Alveolata,alveolata_odb10,eukaryota_odb10,proteins,true,false
```

Comment:
These should reference the GitHub paths similar to samplesheet_test.csv

Reply:
yupyup